AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/AudioSelectionAdapter.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.samples.communication.calling.helpers;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.PopupWindow;
+
+import androidx.annotation.NonNull;
+import androidx.core.content.ContextCompat;
+
+import com.azure.samples.communication.calling.R;
+import com.azure.samples.communication.calling.activities.CallActivity;
+import com.azure.samples.communication.calling.view.BottomCellViewHolder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AudioSelectionAdapter extends BottomCellAdapter {
+
+    private LayoutInflater mInflater;
```

Comment:
Since BottomCellAdapter has this, we don't need AudioSelectionAdapter to hold it

Reply:
Addressed.